src/Middleware/HttpLogging/src/HttpLoggingFields.cs

```diff
@@ -138,12 +138,20 @@ public enum HttpLoggingFields : long
     /// including <see cref="RequestPath"/>, <see cref="RequestProtocol"/>,
     /// <see cref="RequestMethod"/>, and <see cref="RequestScheme"/>.
     /// </summary>
+    /// <remarks>
+    /// The HTTP Request <see cref="HttpRequest.QueryString"/> is not included with this flag as it may contain private information.
+    /// If desired, it should be explicitly specified with <see cref="RequestQuery"/>.
+    /// </remarks>
     RequestProperties = RequestPath | RequestProtocol | RequestMethod | RequestScheme,
 
     /// <summary>
     /// Flag for logging HTTP Request properties and headers.
     /// Includes <see cref="RequestProperties"/> and <see cref="RequestHeaders"/>
     /// </summary>
+    /// <remarks>
+    /// The HTTP Request <see cref="HttpRequest.QueryString"/> is not included with this flag as it may contain private information.
```

Comment:
It's weird to call this out as the reason why query string values are not here when headers are even more likely to contain private information (i.e. Authorize header)

Reply:
The default set of headers has also been audited to make sure it doesn't contain anything w/ private information